src/base/bootstrap-overrides/_base.scss

```diff
@@ -891,3 +891,148 @@ caption {
 	}
 	text-align: center;
 }
+
+
+.p-0 {
```

Comment:
@thekodester Can you please only bring in the Bootstrap 4 classes that you are actually using in your pull request's code? We already have some of the padding classes under "experimental" that we'll graduate to "stable", and we are still brainstorming on the logistic of how we'll make the complete migration from Bootstrap 3 to 4 happen. Thank you!